### ISSUE TITLE
Configure Map Redesign: on/off labels vs switches

### ISSUE THREAD

**Issue by Velomeloman** [osmandapp/OsmAnd#16587]
<table style="width:100%">
  <tr>
  </tr>
  <tr>
    <td><img src="https://user-images.githubusercontent.com/9365308/201366684-70849985-1d7a-4ef0-b5d3-f749784e0651.png"  width="300" height="auto"/> </td>
  </tr>
</table>   

Now there are 2 examples present: 1) MTB 2) Other items

<details>
    <summary>Merged Original request</summary>

There is no uniformity in the Configure map - Routes settings yet. Bicycle and hiking routes have quick access "... |" to the setting, but for some reason the subtitle "Enabled" duplicates the state of the switch. In the Mountain Bike Routes item, the subtitle conveniently displays the current setting, but there is no quick access to this setting, as well as the switch. It is more convenient that the controls are as in the first and third item, and the subtitle as in the second.

![Screenshot_20230303_152828_net.osmand.plus.jpg](https://user-images.githubusercontent.com/109347561/222727832-94671725-2558-4f9d-ac02-ed737a687768.jpg)
![Screenshot_20230303_152711_net.osmand.plus.jpg](https://user-images.githubusercontent.com/109347561/222727890-79585b35-dffe-435c-8dfa-e11b9bd6299a.jpg)

There is no uniformity within the settings either. The option with large buttons and full size descriptions in Bicycle Routes looks more convenient.
</details>

**Comment by vshcherb** [comment]
Spinoff from https://github.com/osmandapp/OsmAnd-Issues/issues/1169

**Comment by Velomeloman** [comment]
@vshcherb broken link. Can't find issue 1169.

**Comment by vshcherb** [comment]
Configure Map requires full redesign (as Configure Screen) - probably concept of quick actions which would allow to have preconfigured switches on top instead of full list of switches that's hard to understand for new users

**Comment by sonora** [comment]
**Side note:** Maybe this time we allow a bit more for **the rules of usability** when we do the redesign. Because for "Configure screen" we made some basic mistakes, and as ca consequence the dialog is not very discoverable now. Here are the weaknesses:

- The first choice a user has to make now when entering the dialog is for what "panel" a configuration shall be performed.
- But in many cases the user is not aware of what is meant by a "panel". And if they are: They do not normally know behind what "panel" you can have which widgets and functionalities. So you end up in a trial and error game.

The better way would be
- to have the choice of **all available functionality** on the top level, i.e. a user can e.g. select if they need a compass displayed, or the street name display, etc.
- Then subsequently, if there is  choice at all, we can let the user decide where on the screen they would like to have that displayed.

In summary, having the "panel" at the top level, obscuring the functionality to a level 2, is a typical programmer view, not the user view...  :wink:

**Comment by dmpr0** [comment]
@sonora 
We can, as soon as we get complains about Configure screen. ;-)

**Comment by vshcherb** [comment]
The better way is arguably better. 

You need to explain the concept as it is and the concept that you have panels to place. All available information will overload user anyway, so you need to introduce grouping. Grouping by places where widgets are displayed is totally fine. 

You probably confusing introduction screen with configuration screen. On introduction screen it's fine to have a page with "what do you want to see" or "how are you going to use the screen". We don't have introduction screens except download maps.

On configuration screen you need to have structure and grouping which is controlled by number of elements.

**Comment by sonora** [comment]
I am certainly not against structure and grouping. All I am saying is that structure we guide the user by should follow the users' priorities, not the codes'. And from a user's perspective the "_What can I have?_" takes priority over the "_Where can I place it?_"
